GroupManager/Controllers/GroupsController.cs

```diff
@@ -84,8 +84,9 @@ public async Task<ActionResult> Index()
         private async Task<string> GetGraphAccessToken(string userId, string[] scopes)
         {
             IConfidentialClientApplication cc = MsalAppBuilder.BuildConfidentialClientApplication();
+            IAccount userAccount = await cc.GetAccountAsync(ClaimsPrincipal.Current.GetMsalAccountId());
 
-            AuthenticationResult result = await cc.AcquireTokenSilent(scopes, ClaimsPrincipal.Current.ToIAccount()).ExecuteAsync(); 
+            AuthenticationResult result = await cc.AcquireTokenSilent(scopes, userAccount).ExecuteAsync(); 
```

Comment:
>AcquireTokenSilent [](start = 51, length = 18)

Clicking on Groups without signing in results in the error "No account or login hint was passed to the AcquireTokenSilent call." I'm not sure if the error was there before

Reply:
According to what I understood, since we are using InMemory cache, when we close the application the cache is cleared. However we still have a cookie for the user in the browser.
Running the app again, automatically logs you in but the cache construction flow was never triggered, resulting in a null return until you sign-out and sign-in again to re-create the memory cache.
Is this a reason for not using InMemory cache on web applications?

Reply:
This problem was solved by catching `MsalUiRequiredException` on `GroupsController.cs`